Orleans.Consensus/Log/ProtobufSerializer.cs

```diff
@@ -0,0 +1,36 @@
+﻿namespace Orleans.Consensus.Log
+{
+    using ProtoBuf.Meta;
+    using System;
+    using System.IO;
+    using System.Threading.Tasks;
+
+    public class ProtobufSerializer<T> : ISerializer<T> where T:new()
+    {
+        RuntimeTypeModel model;
+
+        public ProtobufSerializer(params Type[] subTypes)
```

Comment:
I wonder if RuntimeTypeModel should be injected. Not a change we need to make before merging.